utils/logistics/readLogFiles.js

```diff
@@ -0,0 +1,28 @@
+const fs = require('fs');
+const path = require('path');
+const { SERVER_LOG_DEST } = require('../constants');
+
+const readLogFiles = async (domain, logPath) => {
+  try {
+    const files = fs.readdirSync(logPath);
+    const firstFileData = fs.readFileSync(
+      path.join(
+        logPath,
+        files.filter((filename) => filename.match(/\.json$/))[0]
+      )
+    );
+    const destination = path.join(
+      __dirname,
+      "../../",
+      SERVER_LOG_DEST,
+      domain,
+      JSON.parse(firstFileData.toString()).context.transaction_id
+    );
+    fs.mkdirSync(destination, { recursive: true });
```

Comment:
As discussed yesterday, did we also check to make sure old log files exist there, in case the transaction_id already exists? Make sure merge.json is having all the logs wrt transaction id.

Reply:
Yes, the merged.json will have all the log files for a transaction_id.